roles/bahmni-emr/tasks/main.yml

```diff
@@ -7,18 +7,18 @@
   yum: name="openmrs" state=absent
   ignore_errors: true
 
-- name: Check if bahmni-openmrs rpm needs an update
-  command: yum list installed bahmni-openmrs
-  register: installed_openmrs
-  ignore_errors: true
+# - name: Check if bahmni-openmrs rpm needs an update
+  # command: yum list installed bahmni-openmrs
+  # register: installed_openmrs
+  # ignore_errors: true
 
 # Uninstall 2FA app before uninstalling OpenMRS
 - name: Uninstall two factor authentication rpm
   yum:  name="bahmni-two-factor-auth" state=absent
 
-- name: Uninstall bahmni-openmrs rpm
-  yum: name="bahmni-openmrs" state=absent
-  when: installed_openmrs.stdout.find("{{ bahmni_openmrs_version }}") == -1
+# - name: Uninstall bahmni-openmrs rpm
+  # yum: name="bahmni-openmrs" state=absent
+  # when: installed_openmrs.stdout.find("{{ bahmni_openmrs_version }}") == -1
```

Comment:
After we merge bahmni-openmr, bahmni-emr rpms, do we still need this variable bahmni_openmrs_version? @shilpa-iplit

Reply:
I don't think variable bahmni_openmrs_version is needed, but need to check.
Yes will remove commented lines.

Reply:
Currently the variable bahmni_openmrs_version is defined in 2 files:
1) /bahmni-playbooks/group_vars/dev
2) /bahmni-playbooks/dev/rpm_versions.yml
Its not used anywhere. @SwathiVarkala

Reply:
@shilpa-iplit Please go ahead and remove the unused variable

Reply:
Removed the unused variable bahmni_openmrs_version